generate-test-vector.py

```diff
@@ -1105,14 +1115,141 @@ def generate_unknown_segwit_ver_test():
     # TODO: scanning no taproot output
     # TODO: scanning no inputs for ECDH
 
+def generate_taproot_with_nums_point_test():
+
+    msg = hashlib.sha256(b'message').digest()
+    aux = hashlib.sha256(b'random auxiliary data').digest()
+    G = ECKey().set(1).get_pubkey()
+    outpoints = [
+        ("f4184fc596403b9d638783cf57adfe4c75c605f6356fbc91338530e9831e9e16", 0),
+        ("a1075db55d416d3ca199f55b6084e2115b9345e16c5cf302fc80e9d5fbf5d48d", 0),
+        ("a1075db55d416d3ca199f55b6084e2115b9345e16c5cf302fc80e9d5fbf5d48d", 1)
+    ]
+
+    sender_bip32_seed = 'deadbeef'
+    recipient_bip32_seed = 'f00dbabe'
+    i1, I1 = get_key_pair(0, seed=bytes.fromhex(sender_bip32_seed))
+    i2, I2 = get_key_pair(1, seed=bytes.fromhex(sender_bip32_seed))
+    i3, I3 = get_key_pair(2, seed=bytes.fromhex(sender_bip32_seed))
+
+    if I1.get_y()%2 != 0:
+        i1.negate()
+        I1.negate()
+
+    if I2.get_y()%2 != 0:
+        i2.negate()
+        I2.negate()
+    
+    if I3.get_y()%2 == 0:
+        i3.negate()
+        I3.negate()
+
+    nums_point = [
+        [(i1, True, NUMS_H, True), (i2, True, None, False), (i3, True, NUMS_H, False)],
+        [I1, I2, I3]
+    ]
+
+    test_cases = []
+    comments = [
+        "Single receipient: taproot input with NUMS point"
+    ]
+    for i, inputs in enumerate([nums_point]):
+        sender, recipient, test_case = new_test_case()
+
+        inp = []
+        for x, (key, is_taproot, internal_key, add_annex) in enumerate(inputs[0]):
```

Comment:
is_taproot is always `True` so maybe not needed?

also can remove from the `nums_point` array